package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kuzzle-admin-console",
-  "version": "2.2.4",
+  "version": "2.3.4",
```

Comment:
why do you change the version number manually to release on the development branch?

Reply:
Also, if you want the minor version to be updated, then you should use the `changelog:new-features": this triggers a minor version update from our release tools.

Reply:
I don't want to update the last number as we change it for every hotfix, I want to change the second one. It should be 2.3.0 btw -.-

Reply:
You don't get it: if you manually update the version and release the admin console with our tools, you'll get a "2.3.1" version tag (unless you use `--tag` to force the future version number to 2.3.0).
If you leave the version alone and simply add the `changelog:new-features` label, then the release tools will automatically increment the minor version number.

Reply:
Right, I'm sorry I did some hotfix before and I was used to change the version, forgetting I was making a PR on the development branch -_-